tests/unit/floating_ips/test_client.py

```diff
@@ -168,8 +171,8 @@ def test_get_all(self, floating_ips_client, two_floating_ips_response, params):
         assert bound_floating_ip2.id == 4712
         assert bound_floating_ip2.description == "Web Backend"
 
-    def test_create_with_location(self, floating_ips_client, floating_ip_create_response):
-        floating_ips_client._client.request.return_value = floating_ip_create_response
+    def test_create_with_location(self, floating_ips_client, floating_ip_response):
```

Comment:
We need to test two responses: first with "action" keys, second without "action" key

Reply:
add response check to ```def test_create_with_server```